docs/configuration.rst

```diff
@@ -54,7 +54,11 @@ Exchange Sections
 There is a section for each exchange to configure exchange specific attributes.
 
 - ``all_currencies`` List of all supported currencies for funding. The list have to change only
-  when the exchange adds a new supported currency or removes one.
+  when the exchange adds a new supported currency or removes one. You can create a blacklist of currencies by using '#' in front of a specific one.
```

Comment:
Better grammar:

You can blacklist specific currencies by prefacing it with a #, this is the same as not including it on the list.